README.md

```diff
@@ -22,7 +22,7 @@
 It is based on the Scheme dialect and the R5RS/R7RS specifications. It has extensions to make it easier
 to interact with JavaScript. It work both in the browser and with Node.js.
 
-The name is a recursive acronym which stands for LIPS Is Pretty Simple.
+The name is a recursive acronym which stands for LIPS it's pretty simple.
```

Comment:
This is an acronym it's not an error that it's written this way. LIPS Is Pretty Simple is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym) where the first letter of the word is part of the name.

Reply:
Thanks I didn't know that, i appreciate the heads up i am going to read more on that.